versions/run.noaacloud.ver

```diff
@@ -0,0 +1,10 @@
+export stack_intel_ver=2021.3.0
+export stack_impi_ver=2021.3.0
+export spack_env=gsi-addon-env
+
+export gempak_ver=7.4.2
```

Comment:
```suggestion
```
gempak is not available/supported on AWS/Cloud

Reply:
Removed from AWS/cloud.